### ISSUE TITLE
Function to get Prefix for Datasource

### ISSUE THREAD

**Issue by mkutmon** [bridgedb/BridgeDb#168]
Is it possible to implement a function for datasource to get the prefix?
Currently, you can parse it per identifier using the Xref function getMiriamURN() but it would be nice to just get the prefix for a datasource directly.
 
Example:
```
DataSource ds = DataSource.getExistingBySystemCode("L");
String prefix = ds.getPrefix();
```

Output would be **"ncbigene"**

**Comment by mkutmon** [comment]
```
Xref x = new Xref("1234", DataSource.getExistingBySystemCode("L"));
String urn = x.getMiriamURN();
```

That results in **"urn:miriam:ncbigene:1234"** which contains the prefix but it's a little cumbersome to access it like this.

**Comment by egonw** [comment]
I just implemented this, which will be part of BridgeDb 3.0.5, which I'll release later today (I intend):

```java
DataSource ds = DataSource.getExistingBySystemCode("L");
String prefix = ds.getCompactIdentifierPrefix();
```
